src/test/java/com/revature/services/test/QuestionService.java

```diff
@@ -0,0 +1,14 @@
+package com.revature.services.test;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class QuestionService {
```

Comment:
This test file provides no actual testing. Remove it.

Reply:
code added